ci-operator/step-registry/hypershift/mce/kubevirt/create/hypershift-mce-kubevirt-create-chain.yaml

```diff
@@ -45,12 +52,13 @@ chain:
 
 
         echo "$(date) Creating HyperShift cluster ${CLUSTER_NAME}"
-        /tmp/hypershift create cluster kubevirt ${ICSP_COMMAND} \
+        /tmp/${HYPERSHIFT_NAME} create cluster kubevirt ${ICSP_COMMAND} \
           --name ${CLUSTER_NAME} \
           --node-pool-replicas ${HYPERSHIFT_NODE_COUNT} \
           --memory 16Gi \
           --cores 4 \
           --namespace local-cluster \
+          --release-image ${RELEASE_IMAGE_LATEST} \
           --pull-secret ${PULL_SECRET_PATH}
```

Comment:
for aws, i believe you all are using `odf` as the default storage class. We've had issues with ODF not being able to keep up with etcd writes, so we have been limiting ODF usage to kubevirt VM root volumes, and using the aws csi for etcd.

To do this, when the KubeVirt VMs are running on aws infrastructure, i'd recommend using this cli arg during the cluster create command `--etcd-storage-class="gp3-csi"`. it will help reduce test failures.

Reply:
Thanks @davidvossel, I added it to the command to use this.